### DISCUSSION TITLE
Are there plans to add duckdb support?

### DISCUSSION THREAD

**Discussion by bingocatlion** [diesel-rs/diesel#4344] [Ideas]
Are there plans to add duckdb support?

**Comment by weiznich** [comment]
We do not want to add any new database backends to diesel itself as we have a limited capacity of maintainer time. This is especially true for backends that wouldn't be used by any of the existing maintainers. The way forward for such backends is to implement them as third party backend in their own crate. See [the relevant documentation](https://docs.diesel.rs/2.2.x/diesel/connection/trait.Connection.html#implement-support-for-an-unsupported-database-system) for details about that.

**Comment by nimdeveloper** [comment]
I just started developing a crate for that, check https://github.com/nimdeveloper/better-duck
